34.py

```diff
@@ -0,0 +1,31 @@
+class Solution:
+    def findIndex(self, nums: List[int], target: int, is_first: bool) -> int:
```

Comment:
`is_first` is not descriptive. This variable indicates whether the caller wants to search for the left most or the right most instance of `target` in the `nums` array.